### ISSUE TITLE
Deprecation Warnings when running rspec with commontator installed on fresh rails 6 app (Zeitwerk)

### ISSUE THREAD

**Issue by alaarab** [lml/commontator#160]
On a Fresh Rails 6 App with the Gems rspec-rails and commontator installed, when running rspec the following deprecation warnings come up:

```
alaarab@PanamaJack:~/Sites/myapp$ rspec

The dependency tzinfo-data (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for ruby but the dependency is only for x86-mingw32, x86-mswin32, x64-mingw32, java. To add those platforms to the bundle, run `bundle lock --add-platform x86-mingw32 x86-mswin32 x64-mingw32 java`.
DEPRECATION WARNING: Initialization autoloaded the constants ActionText::ContentHelper and ActionText::TagHelper.

Being able to do this is deprecated. Autoloading during initialization is going
to be an error condition in future versions of Rails.

Reloading does not reboot the application, and therefore code executed during
initialization does not run again. So, if you reload ActionText::ContentHelper, for example,
the expected changes won't be reflected in that stale Module object.

These autoloaded constants have been unloaded.

Please, check the "Autoloading and Reloading Constants" guide for solutions.
 (called from <top (required)> at /mnt/c/Users/Panama Jack/Documents/GitHub/myapp/config/environment.rb:5)
DEPRECATION WARNING: Initialization autoloaded the constants ActionText::ContentHelper and ActionText::TagHelper.

Being able to do this is deprecated. Autoloading during initialization is going
to be an error condition in future versions of Rails.

Reloading does not reboot the application, and therefore code executed during
initialization does not run again. So, if you reload ActionText::ContentHelper, for example,
the expected changes won't be reflected in that stale Module object.

These autoloaded constants have been unloaded.

Please, check the "Autoloading and Reloading Constants" guide for solutions.
 (called from <main> at /mnt/c/Users/Panama Jack/Documents/GitHub/myapp/config/environment.rb:5)
```

**Comment by alaarab** [comment]
I opened up a PR https://github.com/lml/commontator/pull/161 which resolves this issue.

**Comment by Dantemss** [comment]
Thanks. Merged and fixed in v6.2.0.
